packages/css/src/components/dialog/dialog.scss

```diff
@@ -44,6 +44,7 @@
   align-items: flex-start;
   display: flex;
   gap: var(--ams-dialog-header-gap);
+  justify-content: space-between;
 }
 
 @mixin reset {
```

Comment:
Something went wrong with the merge, `ams-dialog__title` and its tokens should be deleted